ocs_ci/helpers/dr_helpers.py

```diff
@@ -559,14 +559,15 @@ def wait_for_all_resources_deletion(
             resource_name=pvc_obj.name, timeout=timeout, sleep=5
         )
 
-    logger.info("Waiting for all PVs to be deleted")
-    sample = TimeoutSampler(
-        timeout=timeout,
-        sleep=5,
-        func=get_pv_count,
-        namespace=namespace,
-    )
-    sample.wait_for_func_value(0)
+    if config.MULTICLUSTER["multicluster_mode"] != "metro-dr":
+        logger.info("Waiting for all PVs to be deleted")
```

Comment:
change this to "Waiting for all PVs to be created"

Reply:
This function `wait_for_all_resources_deletion`  wait for workload and replication resources to be deleted, in this step we are waiting for all PVs are deleted.

Reply:
my bad. this is not an issue.